packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -278,4 +287,223 @@ describe('multi-domain', { experimentalSessionSupport: true, experimentalMultiDo
 
     it('short circuits the secondary domain command queue when "done()" is called early')
   })
+
+  describe('yields', () => {
```

Comment:
Should I break these out to their own file?

Reply:
I'd vote yes.

Reply:
Done